analyses/focal-cn-file-preparation/01-add-ploidy-cnvkit.Rmd

```diff
@@ -26,10 +26,62 @@ histologies_file <- file.path("..", "..", "data", "histologies.tsv")
 histologies_df <- readr::read_tsv(histologies_file, guess_max = 100000)
 ```
 
+# Edits for https://github.com/PediatricOpenTargets/ticket-tracker/issues/177
+```{r}
+# Filter to WXS tumor with no germline sex estimate
+no_estimate <- histologies_df %>% 
+  filter(sample_type == "Tumor" & experimental_strategy == "WXS" &
+           is.na(germline_sex_estimate)) %>%
+  select(-germline_sex_estimate)
+
+# Extract participant ID with no germline sex estimate
+no_estimate_participants <- no_estimate %>% 
+  pull(Kids_First_Participant_ID) %>% 
+  unique()
+
+# Find out matching WGS
+# Use cohort_participant_id for filtering samples that have WGS, when WXS was performed
+no_estimate_participants_match_wgs <- histologies_df %>% 
+  filter(cohort_participant_id %in% no_estimate_participants & 
+           experimental_strategy == "WGS" & sample_type == "Tumor") %>% 
+  select(Kids_First_Participant_ID, germline_sex_estimate)
```

Comment:
this needs to be changed to `Kids_First_Participant_ID`

Reply:
> this needs to be changed to `Kids_First_Participant_ID`

@jharenza I changed it `cohort_participant_id` to `Kids_First_Participant_ID`, but the count for GSE values is still the same. Let me know when you are reviewing this, may be we can jump on a quick call?